src/App.js

```diff
@@ -64,6 +64,11 @@ class App extends Component {
     download(this.canvas);
   };
 
+  shareToTwitter = e => {
+    e.preventDefault();
+    shareToTwitter(this.canvas);
```

Comment:
the naming is confusing :) I think that naming callback should start with `on` prefix, like for example `onShareTwitter`. What do you think?

Reply:
I think so, too :)